### DISCUSSION TITLE
Exclude part of task's description from task text in cardboard?

### DISCUSSION THREAD

**Discussion by ItsFullOfCode** [roovo/obsidian-card-board#110] [Ideas]
Would it be possible, to exclude part of a task's description from showing in Cardboard?
Primary use case: The reminder plugin adds a reminder date/time text string to the task (highlighted in bold):
- [ ] A task with a reminder 🔼 📅 2022-12-28 **(@2022-11-25 23:07)**

In Cardboard this task is presented like this:

![image](https://user-images.githubusercontent.com/53911703/204059943-22074a89-1ee5-4e34-a079-fe66f66940ab.png)

Especially on a board with a large number of tasks the additional date/time text adds clutter without providing useful information.

Would it be possible/feasible to have a board option similar to: "Ignore text after: " _value_
_value_ could be a user-provided string or a predefined choice (for example "(@" in the case of the Reminder plugin)

**Comment by roovo** [comment]
Hiya and thanks!  I've added a note to #94 as it could fit quite nicely into that: namely when I handle dates other than due dates, to think about what I would like to do with instead of just leaving them on the line with the task description.  I am currently talking with @claremacrae on better integration with Tasks and off the back of this I will need to think a bit about other date types so I may tackle it there.

This would only deal specifically with dates appearing on the task lines, which it sounds like would solve your particular issue.  I am aware that there may be a more generic need to hide other stuff too so I will leave it to you (or anyone else) to shout out if you think this is needed too.

Thanks again - I'll update here as thoughts on this progress as I am working with Clare :)

**Reply by ItsFullOfCode** [reply]
Thanks for taking such a feature into consideration!
You are right - your suggested solution would solve my particular scenario and I'd be happy with that. :-)
